internal/provider/resource_hvn_route.go

```diff
@@ -185,15 +185,12 @@ func resourceHvnRouteRead(ctx context.Context, d *schema.ResourceData, meta inte
 		return diag.FromErr(err)
 	}
 
-	idLink, err := parseLinkURL(d.Id(), HVNRouteResourceType)
+	idLink, err := buildLinkFromURL(d.Id(), HVNRouteResourceType, client.Config.OrganizationID)
```

Comment:
There are a few places where parseLinkURL is still being called. Seeing as buildLinkFromURL calls parseLinkURL internally and sets the orgid does it make sense to just replace those calls with buildLinkFromURL?

Reply:
In at least one of these places, the ProjectID from the link is needed to backtrace the OrganizationID, so it isn't known when parseLinkURL is called. Given that, I don't think it hurts to keep parseLinkURL around as a standalone for when the OrganizationID isn't known or isn't wanted/needed in the link.